iron-ajax.html

```diff
@@ -291,7 +291,7 @@
     },
 
     get requestHeaders() {
-      var headers = Object.create(this.headers || {});
+      var headers = this.headers || {};
```

Comment:
Well, we want to make a copy of the object, since we are passing ownership of the object to a different authority (the `core-request` instance). Your other strategy to copy the properties to a new object was a good way to do that.